### ISSUE TITLE
Border on full-with wrapper/section is rendered with default width 600px

### ISSUE THREAD

**Issue by mberhorst** [mjmlio/mjml#693]
My example:
```
<mjml>
    <mj-body>
        <mj-container>
            <mj-section full-width="full-width" background-color="#ffffff" border-bottom="1px solid #ccc">
                <mj-column width="100%">
                    <mj-text text-transform="uppercase" align="center">
                        Text
                    </mj-text>
                </mj-column>
            </mj-section>
        </mj-container>
    </mj-body>
</mjml>
```

Problem: The border is not added to the full-width section, but inside to a 600px wide element.

Is this a bug or am I doing it the wrong way?

**Comment by iRyusa** [comment]
Hi @mberhorst 

This is intended, `full-width` interact only with the background of the section, not it's content. If you want to tweak this you could use something like `css-class` in 3.3.3-beta and custom styling with `mj-style`

**Comment by mberhorst** [comment]
Thanks @iRyusa 
So switching to beta will solve some of my problems.

**Comment by mberhorst** [comment]
Hm I'm now on _3.3.3-beta.2_ but the css-class attribute does not work as aspected. I Added the class to a mj-section but the class is only added to the IE case with an outlook suffix:
```
<!--[if mso | IE]>
      <table role="presentation" border="0" cellpadding="0" cellspacing="0" width="600" align="center" style="width:600px;" class="border-bottom-outlook">

```

**Comment by ngarnier** [comment]
Hey @mberhorst, indeed, there's an issue in the beta where the `css-class` is added on the Outlook conditional comment only if `full-width="full-width` is used (you'll see it's added correctly on the `div` if you remove it). Thanks for reporting, it's fixed for the next release.

**Comment by iRyusa** [comment]
It has been fixed on 3.3.3 release so I can close this issue 👍
